### PR TITLE
feat: French translation

### PR DESCRIPTION
**Pull request by darahak**
Built and launched a local version.
Seems to work okay except some texts that were not in `src\_data\sites\en.yml`, so they remain in English.

### CODE REVIEW

**Comment by nzakas** [comment]
Thanks! It looks like there are a couple of commits that maybe @Brack0 worked on? Those commits aren't properly tied to a GitHub ID, which is why the CLA check is failing. @Brack0 can you update those commits and sign the CLA?

File: src/_data/sites/fr.yml [hunk 1]
```diff
@@ -0,0 +1,599 @@
...
+#------------------------------------------------------------------------------
+# Global Settings
+#------------------------------------------------------------------------------
+
+language:
+  code: fr
+  flag: 🇫🇷
+  name: French
+locale: fr-FR
+hostname: new.eslint.org
```
Comment (btmills) [inline src/_data/sites/fr.yml:17]:
```suggestion
hostname: fr.eslint.org
```

**Comment by nzakas** [comment]
@darahak it looks like we are stuck on those missing commits. Would you be able to remove them and resubmit?

**Comment by darahak** [comment]
@nzakas I'll have a look this weekend.

**Comment by darahak** [comment]
@nzakas I amended the faulty commits with the right user ID, but it's still failing.
@Brack0 Did you sign the CLA?

**Comment by Brack0** [comment]
> @Brack0 Did you sign the CLA?

Yep, should be good now.

File: src/_data/sites/fr.yml [hunk 2]
```diff
@@ -0,0 +1,599 @@
+#------------------------------------------------------------------------------
+# English Site Details
+# The primary ESLint site that is hosted at eslint.org
+# Author: Nicholas C. Zakas
```
Comment (snitin315) [inline src/_data/sites/fr.yml:4]: You should add your name here.

File: src/_data/sites/fr.yml [hunk 3]
```diff
@@ -0,0 +1,599 @@
...
+  name: French
+locale: fr-FR
+hostname: fr.eslint.org
+
+#------------------------------------------------------------------------------
+# Analytics
+#------------------------------------------------------------------------------
+
+google_analytics:
+  code: "UA-60915033-1"
```
Comment (snitin315) [inline src/_data/sites/fr.yml:24]:
```suggestion
  code: ""
```

File: src/_data/sites/fr.yml [hunk 4]
```diff
@@ -0,0 +1,599 @@
...
+
+#------------------------------------------------------------------------------
+# Ads
+#------------------------------------------------------------------------------
+
+carbon_ads:
+  serve: CESDV2QM
+  placement: eslintorg
```
Comment (snitin315) [inline src/_data/sites/fr.yml:32]:
```suggestion
  serve: ""
  placement: ""
```

File: src/_data/sites/fr.yml (hunk 2 above)
Comment (nzakas) [inline src/_data/sites/fr.yml:4]:
```suggestion
# French Site Details
# The French ESLint site that is hosted at fr.eslint.org
# Author: Dara Hak
```

**Review by nzakas — APPROVED** [review]

**Comment by nzakas** [comment]
Thanks! Everything looks good, so I'll merge this and we can handle updates in separate PRs as necessary.

**Comment by nzakas** [comment]
@Brack0 can you send us an email at contact (at) eslint (dot) org so we can pay you for this contribution? :)

(I already have @darahak's email address)

Thanks!

**Pull request merged**